docs/reference/sql/language/syntax/commands/select.asciidoc

```diff
@@ -371,26 +371,52 @@ all are equally relevant.
 [[sql-syntax-limit]]
 ==== LIMIT
 
-The `LIMIT` clause restricts (limits) the number of rows returns using the format:
+The `LIMIT` clause restricts (limits) the number of rows returned using the format:
 
 [source, sql]
 ----
-LIMIT ( count | ALL )
+LIMIT ( <count> | ALL )
 ----
 
 where
 
-count:: is a positive integer or zero indicating the maximum *possible* number of results being returned (as there might be less matches than the limit). If `0` is specified, no results are returned.
+count:: is a positive integer or zero indicating the maximum *possible* number of results being returned (as there might be less matches
+than the limit). If `0` is specified, no results are returned.
 
 ALL:: indicates there is no limit and thus all results are being returned.
 
-To return 
-
 [source, sql]
 ----
 include-tagged::{sql-specs}/docs/docs.csv-spec[limitBasic]
 ----
 
+[NOTE]
+<<sql-syntax-top, `TOP`>> and <<sql-syntax-limit, `LIMIT`>> cannot be used in combination in the same query and error is returned.
+
+
+[[sql-syntax-top]]
+==== TOP
+
+The `TOP` clause restricts (limits) the number of rows returned using the format:
+
+[source, sql]
+----
+SELECT TOP <count> <list of fields> ...
+----
+
+where
+
+count:: is a positive integer or zero indicating the maximum *possible* number of results being returned (as there might be less matches
+than the limit). If `0` is specified, no results are returned.
+
+[source, sql]
+----
+include-tagged::{sql-specs}/docs/docs.csv-spec[topBasic]
+----
+
+[NOTE]
+<<sql-syntax-top, `TOP`>> and <<sql-syntax-limit, `LIMIT`>> cannot be used in combination in the same query and error is returned.
```

Comment:
`...cannot be used together in the same query and an error is returned otherwise` sounds better? If so, same sentence was used for `LIMIT` and the change is needed there as well.